### ISSUE TITLE
Checksum issue

### ISSUE THREAD

**Issue by hunterleebrown-cbs** [AppsFlyerSDK/segment-appsflyer-ios#143]
A new developer I'm onboarding is getting this Swift Package Issue:

`checksum of downloaded artifact of binary target 'AppsFlyerLib' (f00873e9c3764d82ae87326a0280a361871dcde4e7a338cb29b555e85c943764) does not match checksum specified by the manifest (53c824272e8729967af6b22e013577eccbc6e412b7e2da2805ddf3c1ecf7cfc5)`

If you look here at 6.14.3 of AppsFlyer.lib
https://github.com/AppsFlyerSDK/AppsFlyerFramework/blob/6.14.3/Package.swift

`    targets: [
        .binaryTarget(
            name: "AppsFlyerLib",
            url: "https://github.com/AppsFlyerSDK/AppsFlyerFramework/releases/download/6.14.3/AppsFlyerLib-Static-SPM.xcframework.zip",
            checksum: "53c824272e8729967af6b22e013577eccbc6e412b7e2da2805ddf3c1ecf7cfc5"
        )
    ]`
    
Is there some misalignment between this project and AppsFlyerLib?     Can we jump to 6.14.5. ?  Can the checksum be checked in the Package.swift to makesure it's correct in the AppsFlyerFramework repo?

**Comment by af-obodovskyi** [comment]
@hunterleebrown-cbs please try this solution 
```
rm -rf ~/Library/Caches/org.swift.swiftpm
rm -rf ~/Library/org.swift.swiftpm
rm -rf ~/.swiftpm

Also delete derived data and optionally delete Package.resolved file.
```

**Comment by hunterleebrown-cbs** [comment]
> @hunterleebrown-cbs please try this solution
> 
> ```
> rm -rf ~/Library/Caches/org.swift.swiftpm
> rm -rf ~/Library/org.swift.swiftpm
> rm -rf ~/.swiftpm
> 
> Also delete derived data and optionally delete Package.resolved file.
> ```

Yes, we've done all of that.   And which Package.resolved file to delete?

**Comment by hunterleebrown-cbs** [comment]
Also is the Packages.resolved file correct here in this repo?  6.9.1 looks old.

`      {
        "package": "AppsFlyerLib",
        "repositoryURL": "https://github.com/AppsFlyerSDK/AppsFlyerFramework.git",
        "state": {
          "branch": null,
          "revision": "7a3cd712a77b30e487e53102b101c5e9ff96a95e",
          "version": "6.9.1"
        }`

**Comment by hunterleebrown-cbs** [comment]
Closing issue.  Corporate ware on the new employee's machine (a contractor) was blocking proper access.
